flink-connectors/flink-sql-connector-hbase-1.4/src/main/resources/META-INF/NOTICE

```diff
@@ -8,14 +8,11 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.google.guava:guava:12.0.1
 - com.yammer.metrics:metrics-core:2.2.0
 - commons-codec:commons-codec:1.13
-- commons-configuration:commons-configuration:1.7
```

Comment:
the shade plugin contains an explicit include for this dependency

Reply:
Good point! Missed it. Fixing.